.circleci/config.yml

```diff
@@ -41,7 +41,7 @@ parameters:
     default: /tmp/wordpress-tests-lib
   wp_core_dir:
     type: string
-    default: /tmp/wordpress/
+    default: /tmp/wordpress
```

Comment:
Why are we changing this?

Reply:
Actually, for no particular reason… The scripts expect the unslashed path, and with the trailing slash it turns into smth like `/tmp/wordpres//something`. It works but does not look nice.

Reply:
If that's the case, it will indeed look better.